src/components/common/YourCompany/YourCompany.vue

```diff
@@ -179,10 +179,10 @@
       </v-row>
     </div>
 
-    <v-divider v-if="isChangeRegFiling || isFirmConversionFiling" class="mx-4 my-1" />
+    <v-divider v-if="isChangeRegFiling || isFirmConversionFiling || isFirmCorrectionFiling" class="mx-4 my-1" />
```

Comment:
These 3 are types of firm filings, too, so you can use a single getter if you want.